<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>80</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="output">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>41</width>
     <height>9</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_sort">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>41</width>
     <height>9</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="by_times">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>80</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="by_date">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>80</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
